contracts/StMATIC.sol

```diff
@@ -201,12 +201,10 @@ contract StMATIC is
                 if (lastWithdrawnValidatorId > operatorInfosLength - 1) {
                     lastWithdrawnValidatorId = 0;
                 }
+                address validatorShare = operatorInfos[lastWithdrawnValidatorId]
+                    .validatorShare;
 
-                address stakedAmount = operatorInfos[
```

Comment:
The name stakedAmount was recommended in the audit, I think. Although it was just a comment, not a major issue.

Reply:
I saw that but this values is the address of the validatorShare contract.

Reply:
The comment regarding validatorShare -> stakedAmount was aimed at this line: https://github.com/Shard-Labs/PoLido/blob/bda8645bff4916a12d21fff38df8379e036724fd/contracts/StMATIC.sol#L317